libcloud/compute/drivers/openstack.py

```diff
@@ -120,6 +120,7 @@ class OpenStackNodeDriver(NodeDriver, OpenStackDriverMixin):
         'PASSWORD': NodeState.PENDING,
         'RESCUE': NodeState.PENDING,
         'REBOOT': NodeState.REBOOTING,
+        'RESIZE': NodeState.MIGRATING,
```

Comment:
I'll make a small change so it uses ``reconfiguring`` state instead, but besides that, LGTM.

Reply:
Also, I didn't actually know it utilizes states from OpenStack driver.

Technically, this change should be safe either way so it's fine to add it here.